index.html

```diff
@@ -405,40 +433,41 @@ <h2 id="format">The did:orb Format</h2>
 
         <pre>
           did-orb-format        = "did:orb:" (orb-canonical-did / orb-scheme-did / unpublished-did)
-          orb-canonical-did     = min-graph-cid ":" did-suffix
-          orb-scheme-did        = (dht-scheme / webcas-scheme) ":" orb-canonical-did
-          unpublished-did       = "interim:" reg-name ":" did-suffix [":" long-form-suffix-data]
-          dht-scheme            = ( "ipfs" )
-          webcas-scheme         = "webcas:" reg-name
+          orb-canonical-did     = min-graph-multihash ":" did-suffix
+          orb-scheme-did        = (hashlink-scheme / hostmeta-scheme)
+          hashlink-scheme       = "hl:" min-graph-multihash ":" graph-hl-metadata ":" did-suffix
+          hostmeta-scheme       = ("https" / "ipns") ":" reg-name ":" orb-canonical-did
           reg-name              = 1*idchar                ; more constrained than [RFC3986]
-          min-graph-cid         = 10*idchar               ; a short 3rd segment is reserved for schemes.
+          min-graph-multihash   = 10*idchar               ; a short 3rd segment is reserved for schemes.
+          graph-hl-metadata     = 1*idchar
           did-suffix            = 1*idchar
           long-form-suffix-data = 1*idchar                ; only applicable in the local-scheme
         </pre>
 
         <p>
           See [[?RFC3986]] for the original definition of reg-name and [[DID-CORE]] for the definition of idchar.
           [[SIDETREE]] provides additional explanation for the <i>did-suffix</i> and <i>long-form-suffix-data</i> elements.
+          [[MULTIHASH]] and [[MULTIBASE]] defines the multihash and multibase formats.
         </p>
 
         <pre class="example" title="A canonical Orb DID">
-          did:orb:bafkreiatkubvbkdidscmqynkyls3iqawdqvthi7e6mbky2amuw3inxsi3y:EiDyOQbbZAa3aiRzeCkV7LOx3SERjjH93EXoIM3UoN4oWg
+          did:orb:uEiDlXjleTwr4eZalpXVy086zs-TPK-h54ojbpl7EBvZeHQ:EiDyOQbbZAa3aiRzeCkV7LOx3SERjjH93EXoIM3UoN4oWg
         </pre>
 
-        <pre class="example" title="An Orb DID that uses the WebCAS scheme for content discovery">
-          did:orb:webcas:example.com:bafkreiatkubvbkdidscmqynkyls3iqawdqvthi7e6mbky2amuw3inxsi3y:EiDyOQbbZAa3aiRzeCkV7LOx3SERjjH93EXoIM3UoN4oWg
+        <pre class="example" title="An Orb DID that uses the Hashlink scheme for content discovery">
+          did:orb:hl:uEiDlXjleTwr4eZalpXVy086zs-TPK-h54ojbpl7EBvZeHQ:uoQ-Bc2h0dHBzOi8vZXhhbXBsZS5jb20:EiDyOQbbZAa3aiRzeCkV7LOx3SERjjH93EXoIM3UoN4oWg
```

Comment:
We should consider having the multibase prefix (`u`) on the Sidetree suffix too.